openapi2conv/openapi2_conv.go

```diff
@@ -364,6 +370,7 @@ func FromV3Swagger(swagger *openapi3.Swagger) (*openapi2.Swagger, error) {
 		if pathItem == nil {
 			continue
 		}
+		result.AddOperation(path, "GET", nil)
```

Comment:
What does this solve?

Reply:
Some of the swagger files I was converting have paths defined with params but no operations, im not sure if that is even sensible swagger, but adding this call means that the call to `result.Paths[path].Parameters = params` is guaranteed safe, without having to check that the maps are initialised. The nil "GET" op becomes a no-op anyway

Reply:
This project is meant to support only the spec. Please fix your swagger files so they are valid and revert this.